google/cloud/security/scanner/scanners/cloudsql_rules_scanner.py

```diff
@@ -42,10 +44,10 @@ def _get_project_policies(self):
             dict: If successful returns a dictionary of project policies
         """
         project_policies = {}
-        project_policies = (
-            project_dao.ProjectDao().get_project_policies('projects',
-                                                          self.\
-                                                          snapshot_timestamp))
+        project_policies = (project_dao
+                            .ProjectDao(self.global_configs)
```

Comment:
There's no way to break this line well (unless we separate the instantiation from the method call). Just be consistent, for instance `bucket_rules_scanner.py` in this PR is different.